pkg/engine2/testdata/ecs_rds.dataflow-viz.yaml

```diff
@@ -12,6 +12,6 @@ resources:
     parent: vpc/vpc-0
 
   ecs_service/ecs_service_0 -> rds_instance/rds-instance-2:
-    path: aws:ecs_task_definition:ecs_service_0,aws:iam_role:ecs_service_0-rds-instance-2
+    path: aws:ecs_task_definition:ecs_service_0,aws:iam_role:ecs_service_0-execution-role
```

Comment:
👍 fixed the bug w/ path expand vertex dependencies so this gets set by the op rule instead of expansion